docs/en/loading/Flink-connector-starrocks.md

```diff
@@ -104,7 +105,7 @@ In your Maven project's `pom.xml` file, add the Flink connector as a dependency
 | sink.max-retries                  | No           | 3                 | The number of times that the system retries to perform the Stream Load job. This parameter is available only when you set `sink.version` to `V1`. Valid values: 0 to 10.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
-| sink.connect.timeout-ms           | No           | 1000              | The timeout for establishing HTTP connection. Valid values: 100 to 60000. Unit: ms.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| sink.connect.timeout-ms           | No           | 30000              | The timeout for establishing HTTP connection. Valid values: 100 to 60000. Unit: ms.  Before Flink connector v1.2.9, the default value is 1000.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/80975f0d-f634-4965-acac-6e2a80325786)
Remove extra space preceding "Before", and wrap "1000" in code format.